components/script/dom/webidls/Location.webidl

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // https://html.spec.whatwg.org/multipage/#location
-[Exposed=Window, Unforgeable] interface Location {
+[Unforgeable] interface Location {
```

Comment:
The default is `Exposed=Window`, so this doesn't have any effect, except for making the code a little less readable. I would prefer if you reverted these changes.